### PR TITLE
Add docs for account invite endpoint

### PR DESCRIPTION
**Pull request by shadowhand**
Refs WEB-3300

### CODE REVIEW

File: source/methods/accounts.md.erb [hunk 1]
```diff
@@ -118,3 +118,51 @@ Key | Description
...
+### Parameters
+
+Either `email` or `phone_number` must be provided. If both are provided, an SMS and an email will be sent.
+
+Key | Description
+--- | -----------
+<% param "account_id" do %>ID of the account object.<% end %>
+<% param "email" do %>Email address of the manager.<% end %>
+<% param "phone_number" do %>Phone number of the manager.<% end %>
+<% param "region" do %>Two letter country code of the phone number. This can be omitted if the phone number is in E164 format.<% end %>
```
Comment (ameech) [inline source/methods/accounts.md.erb]: Can we link to the E164 format? Or is that already covered somewhere else?

File: source/methods/accounts.md.erb [hunk 2]
```diff
@@ -118,3 +118,53 @@ Key | Description
...
+### Parameters
+
+Either `email` or `phone_number` must be provided. If both are provided, an SMS and an email will be sent.
+
+Key | Description
+--- | -----------
+<% param "account_id" do %>ID of the account object.<% end %>
+<% param "email" do %>Email address of the manager.<% end %>
+<% param "phone_number" do %>Phone number of the manager.<% end %>
+<% param "region" do %>Two letter country code of the phone number. This can be omitted if the phone number is in [E164][e164] format.<% end %>
```
Comment (shadowhand) [inline source/methods/accounts.md.erb]: @ameech added link.

Comment (ameech) [inline source/methods/accounts.md.erb]: :+1:

**Comment by ameech** [comment]
:+1:

**Comment by shadowhand** [comment]
@dolfelt are these docs still valid after your changes?

**Comment by dolfelt** [comment]
@shadowhand Yes. I didn't change any of the parameters.

**Comment by shadowhand** [comment]
@dolfelt okay then it just needs a rebase.

**Pull request merged**